content/doc/book/pipeline/syntax.adoc

```diff
@@ -134,6 +134,8 @@ need to contain its own `agent` section. For example: `agent none`
 
 label:: Execute the Pipeline, or stage, on an agent available in the Jenkins
 environment with the provided label. For example: `agent { label 'my-defined-label' }`
++
+Label conditions can also be used. For example: `agent { label 'my-label1 && my-labe2' }` or `agent { label 'my-label1 || my-labe2' }`
```

Comment:
Complete the second word in the label?
```suggestion
Label conditions can also be used. For example: `agent { label 'my-label1 && my-label2' }` or `agent { label 'my-label1 || my-label2' }`
```

Reply:
I will merge this tomorrow whether you take the suggested wording change or not.

Thanks again for the pull request!

Reply:
Would be better to integrate the change, no need to introduce new typos. But we can do it as copy editors on the top of the pull request

Reply:
Of course, typos should be fixed. I applied it.